library/network_connections.py

```diff
@@ -222,31 +222,29 @@ def link_infos(cls, refresh=False):
         return linkinfos
 
     @classmethod
-    def link_info_find(cls, refresh=False, mac=None, ifname=None):
```

Comment:
The first line of the commit message is unnecessarily long, please make it shorter.

Reply:
I rewrote it as `refactor: Make SysUtil.link_info_find() more pythonic`.  Thanks.